### PR TITLE
feat: add `/swagger.json` handler

### PR DESCRIPTION
**Pull request by babakks**
This PR adds the `/swagger.json` handler.

Fixes CSS-7326

### CODE REVIEW

File: rebac-admin-backend/v1/swagger.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+)
+
+// Returns the OpenAPI spec as a JSON file.
+// (GET /swagger.json)
+func (h handler) SwaggerJson(w http.ResponseWriter, req *http.Request) {
+	swagger, err := resources.GetSwagger()
+	if err != nil {
+		writeErrorResponse(w, NewUnknownError("cannot retrieve swagger instance"))
```
Comment (alesstimec) [inline rebac-admin-backend/v1/swagger.go:17]: suggestion: s/instance/data/

File: rebac-admin-backend/v1/swagger.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		return
+	}
+
+	body, err := swagger.MarshalJSON()
+	if err != nil {
+		writeErrorResponse(w, NewUnknownError("cannot marshal spec as JSON"))
+		return
+	}
+
+	w.Write(body)
```
Comment (alesstimec) [inline rebac-admin-backend/v1/swagger.go:27]: would we want to log in case Write returns an error.. :shrug:

**Review by alesstimec — APPROVED** [review]
LGTM with two suggestions

File: rebac-admin-backend/v1/swagger.go (hunk 2 above)
Comment (babakks) [reply rebac-admin-backend/v1/swagger.go:27]: Actually I had that in my mind, but we haven't yet selected a logging package/implementation for the library. I think for now, I can just use the standard logger and we can change it, if needed, in the future.

**Review by BarcoMasile — APPROVED** [review]
LGTM

**Pull request merged**